dockerfiles/README.md

```diff
@@ -190,8 +188,7 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
     - Note that if OpenVINO was installed with root permissions, this file has to be changed with the same permissions.
 5. Run the docker image by mounting the device drivers
     ```
-    docker run -it --rm --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb --mount type=bind,source=/var/tmp,destination=/var/tmp --device /dev/ion:/dev/ion  onnxruntime-vadm:latest
-
+    docker run -itu root:root --rm --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb --mount type=bind,source=/var/tmp,destination=/var/tmp --device /dev/ion:/dev/ion  onnxruntime-vadm:latest
```

Comment:
Are cgroup rules only needed for vadm? What about Myriad_fp16?

Reply:
Yes cgroup rule is required for Myriad_fp16 as well and this is already documented here.

**3. Run the docker image by mounting the device drivers
    ```
    docker run -it --rm --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb onnxruntime-myriad:latest
    ```**
  
Refer here:
https://github.com/microsoft/onnxruntime/blob/c08433857ba7e67be3e67ea630326e0bf31f5e76/dockerfiles/README.md#openvino-on-myriad-vpu-accelerator